pages/project/read/content/index.js

```diff
@@ -106,10 +138,21 @@ module.exports = merge({}, baseContent, {
   },
   manageAccess: {
     title: 'Guest access',
-    content: 'Allow colleagues to view this project. Administrators will have access by default.',
+    content: 'Allow colleagues to view this application. Administrators will have access by default.',
     action: 'Grant access'
   },
-  related: {
-    nts: 'Non-technical summary'
+  otherDocuments: {
+    heading: 'Other documents',
+    links: {
+      nts: 'Non-technical summary'
+    }
+  },
+  manage: {
+    noPermissions: 'You don\'t have permission to amend, revoke or grant other users access to this licence.'
+  },
+  changeLicenceHolder: {
+    title: 'Change licence holder',
+    description: 'Change the prospective licence holder of this project.',
```

Comment:
Is this content still appropriate for granted licences?

Reply:
This prompted me to fix a bug - this panel should only display for applications, not active licences. The CTA for change licence holder on active licences is in the start amendment section instead, and uses different content.